src/main/java/org/takes/facets/auth/social/PsGithub.java

```diff
@@ -87,7 +109,7 @@ public PsGithub(final String gapp, final String gkey) {
             throw new IllegalArgumentException("code is not provided");
         }
         return Collections.singleton(
-            PsGithub.fetch(this.token(href.toString(), code.next()))
+            this.fetch(this.token(href.toString(), code.next()))
```

Comment:
@yegor256 Is PsGithub used in any real project?
I doubt about this line. I think will be better allow to set the `redirect_uri` in the ctor. `redirect_uri` is not mandatory parameter and it  must reference a subdirectory of the callback URL(from Github application's settiings).
If we make it optional whether it will not affect existing applications?